### PR TITLE
🍔 Menus

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

File: temba/contacts/views.py [hunk 1]
```diff
@@ -870,13 +858,24 @@ def build_content_menu(self, menu):
...
+                        f"{reverse('flows.flow_broadcast')}?c={obj.uuid}",
+                        as_button=self.is_spa(),
+                        disabled=True,
+                    )
                 if self.has_org_perm("contacts.contact_open_ticket"):
                     menu.add_modax(
                         _("Open Ticket"), "open-ticket", reverse("contacts.contact_open_ticket", args=[obj.id])
                     )
+
+                if len(menu.groups[0]) > 0:
```
Comment (rowanseymour) [inline temba/contacts/views.py:876]: you don't need this - `ContentMenu.as_items()` ignores empty groups

File: temba/orgs/tests.py [hunk 2]
```diff
@@ -3582,14 +3582,14 @@ def test_workspace(self):
             parent=self.org,
         )
 
-        with self.assertNumQueries(48):
+        with self.assertNumQueries(58):
```
Comment (rowanseymour) [inline temba/orgs/tests.py:3585]: what are these?

File: temba/contacts/views.py (hunk 1 above)
Comment (ericnewcomer) [reply temba/contacts/views.py:876]: This isn't ready to call as_items yet though. It's from only wanting a divider if an item has already been added (which may not be depending on permissions, etc)

File: temba/orgs/tests.py (hunk 2 above)
Comment (ericnewcomer) [reply temba/orgs/tests.py:3585]: Added dtone, prometheus, etc, to the workspace page.

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**